File: src/main/java/com/campool/model/RentalInfo.java
package com.campool.model;

import com.campool.enumeration.RentalStatus;
import java.time.LocalDate;
import lombok.Getter;
import lombok.RequiredArgsConstructor;
import lombok.ToString;

@Getter
@RequiredArgsConstructor
@ToString
public class RentalInfo {

    private final int rentalId;

    private final String title;

    private final String description;

    private final RentalStatus status;

    private final String userId;

    private final LocalDate startDate;

    private final LocalDate endDate;

    private final int cost;

    private final double longitude;

    private final double latitude;

}

File: src/main/java/com/campool/model/RentalInfo.kt
package com.campool.model

import com.campool.enumeration.RentalStatus
import java.time.LocalDate

data class RentalInfo(
    val rentalId: Long,
    val title: String,
    val description: String,
    val status: RentalStatus,
    val userId: String,
    val startDate: LocalDate,
    val endDate: LocalDate,
    val cost: Int,
    val longitude: Double,
    val latitude: Double
)